include/cantera/base/logger.h

```diff
@@ -70,7 +70,7 @@ class Logger
     //! Write a warning message.
     /*!
      * The default behavior is to write to the logging output.
-     * @param warning  String specifying type of warning; @see Logger::warn
+     * @param warning  String specifying type of warning; See Logger::warn
```

Comment:
```suggestion
     * @param warning  String specifying type of warning
```
This is the docstring for `Logger::warn`